src/print.jl

```diff
@@ -130,7 +130,17 @@ function var_str(::Type{IJuliaMode}, v::AbstractVariableRef; mathmode=true)
         return math("noname", mathmode)
     end
 end
-
+# We want arrays of variables to be printed with `JuMP.VariableRef` as eltype
```

Comment:
Eh, this is another chunk of code that will break between julia versions. Weird printing will also confuse developers when they try to create arrays to store `VariableRef`s.

Reply:
Yes, but it improves printing :-P Ok I'll revert this ^^